pkg/cli-runtime/testing/validatable_table.go

```diff
@@ -98,5 +117,12 @@ func (tc ValidatableTestCase) Run(t *testing.T) {
 		if !tc.ShouldValidate && len(tc.ExpectFieldErrors) == 0 {
 			t.Error("one of ShouldValidate=true or ExpectFieldErrors is required")
 		}
+
+		if tc.CleanUp != nil {
+			var err error
+			if err = tc.CleanUp(ctx, t); err != nil {
+				t.Errorf("unexpected error in clean up: %v", err)
+			}
```

Comment:
```suggestion
			if err := tc.CleanUp(ctx, t); err != nil {
				t.Errorf("unexpected error in clean up: %v", err)
			}
```

Reply:
ok, changed

Reply:
I cant see the change yet. Not sure if I am missing something